lib/GHCup/GHC.hs

```diff
@@ -1035,6 +1043,24 @@ compileGHC targetGhc crossTarget vps bstrap hghc jobs mbuildConfig patches aargs
     pure installVer
 
  where
+  bootAndGenVersion :: ( MonadReader env m
+                      , HasSettings env
+                      , HasDirs env
+                      , HasLog env
+                      , MonadIO m
+                      , MonadThrow m
+                      )
+                   => GHCupPath
+                   -> Excepts '[ProcessError, NotFoundInPATH] m ()
+  bootAndGenVersion tmpUnpack = do
+    let bootFile = fromGHCupPath tmpUnpack </> "boot"
+    hasBootFile <- liftIO $ doesFileExist bootFile
+    when hasBootFile $ do
```

Comment:
I'm not sure. As in: if we have booted, does it mean there is always a `VERSION` file? From what I understand one has to run configure too.

@bgamari @andreasPK